.github/workflows/main.yml

```
@@ -17,6 +17,7 @@ jobs:
     name: Test
     steps:
       - uses: actions/checkout@v4.1.0
-
-      - name: No-op
-        run: echo "no-op"
+      - name: Unit Tests
```

We should print the node and npm versions to make it easier for future us.

```suggestion
      - name: Diagnostics
        run: |
          node --version
          npm --version
      - name: Unit Tests
```